test/core/simd/simd_i16x8_arith2.wast

```diff
@@ -16,16 +17,18 @@
   (func (export "i16x8.max_u_with_const_7") (result v128) (i16x8.max_u (v128.const i16x8 0 0 1 1 2 2 3 3) (v128.const i16x8 3 3 2 2 1 1 0 0)))
   (func (export "i16x8.avgr_u_with_const_8") (result v128) (i16x8.avgr_u (v128.const i16x8 -32768 -32768 32767 32767 16384 16384 65535 65535) (v128.const i16x8 65535 65535 16384 16384 32767 32767 -32768 -32768)))
   (func (export "i16x8.avgr_u_with_const_9") (result v128) (i16x8.avgr_u (v128.const i16x8 0 0 1 1 2 2 3 3) (v128.const i16x8 3 3 2 2 1 1 0 0)))
-  (func (export "i16x8.min_s_with_const_10") (param v128) (result v128) (i16x8.min_s (local.get 0) (v128.const i16x8 -32768 -32768 32767 32767 16384 16384 65535 65535)))
-  (func (export "i16x8.min_s_with_const_11") (param v128) (result v128) (i16x8.min_s (local.get 0) (v128.const i16x8 0 0 1 1 2 2 3 3)))
-  (func (export "i16x8.min_u_with_const_12") (param v128) (result v128) (i16x8.min_u (local.get 0) (v128.const i16x8 -32768 -32768 32767 32767 16384 16384 65535 65535)))
-  (func (export "i16x8.min_u_with_const_13") (param v128) (result v128) (i16x8.min_u (local.get 0) (v128.const i16x8 0 0 1 1 2 2 3 3)))
-  (func (export "i16x8.max_s_with_const_14") (param v128) (result v128) (i16x8.max_s (local.get 0) (v128.const i16x8 -32768 -32768 32767 32767 16384 16384 65535 65535)))
-  (func (export "i16x8.max_s_with_const_15") (param v128) (result v128) (i16x8.max_s (local.get 0) (v128.const i16x8 0 0 1 1 2 2 3 3)))
-  (func (export "i16x8.max_u_with_const_16") (param v128) (result v128) (i16x8.max_u (local.get 0) (v128.const i16x8 -32768 -32768 32767 32767 16384 16384 65535 65535)))
-  (func (export "i16x8.max_u_with_const_17") (param v128) (result v128) (i16x8.max_u (local.get 0) (v128.const i16x8 0 0 1 1 2 2 3 3)))
-  (func (export "i16x8.avgr_u_with_const_18") (param v128) (result v128) (i16x8.avgr_u (local.get 0) (v128.const i16x8 -32768 -32768 32767 32767 16384 16384 65535 65535)))
-  (func (export "i16x8.avgr_u_with_const_19") (param v128) (result v128) (i16x8.avgr_u (local.get 0) (v128.const i16x8 0 0 1 1 2 2 3 3)))
+  (func (export "i16x8.abs_with_const_10") (result v128) (i16x8.abs (v128.const i16x8 -32768 -32768 32767 32767 16384 16384 65535 65535)))
+  (func (export "i16x8.abs_with_const_11") (result v128) (i16x8.abs (v128.const i16x8 65535 65535 16384 16384 32767 32767 -32768 -32768)))
```

Comment:
I'm not sure it makes sense to export multiple "abs_with_const" functions, since abs only takes a single argument anyway. We could instead just export a single "abs" function then pass it different constants in the asserts.

Reply:
Indeed, I will remove one of them.

Reply:
Done.

Reply:
I'm still not sure what the benefit of the remaining "abs_with_const" test is. Is there precedence in other spec tests for exporting a function applying a unary op to a const?

Reply:
We have similar tests at https://github.com/WebAssembly/simd/blob/master/test/core/simd/simd_f32x4.wast#L29, which was suggested by @AndrewScheidecker 
 at https://github.com/WAVM/WAVM/pull/194#issuecomment-534553081, sorry I couldn't remember the benefit of this checkpoint, @AndrewScheidecker, could you help clarification this?

Reply:
For WAVM, the function body is optimized, but the combination of the function body with the constant argument from the invoke is not. I assume that's true in general for other runtimes, though maybe those that use the test scripts by translating them to JavaScript have the potential to inline the call that the invoke is translated to.

Putting a constant in the function body is meant to test that there aren't any bugs in constant folding, or the optimizer in general.

Reply:
@AndrewScheidecker, thank you!  @tlively, is that answer your concern?